.github/workflows/test.yml

```diff
@@ -15,10 +15,17 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v2
-      - name: Use Node.js 14.x
+      -
+        name: Read Node.js version (Unix)
+        id: nodejs-version
+        run: |
+          content=`cat ./.nvmrc`
+          echo "::set-output name=content::$content"
+      - name: Setup Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 14.x
+          # See https://github.com/actions/setup-node/issues/32
+          node-version: ${{ steps.nodejs-version.outputs.content }}
```

Comment:
The one drawback I see from making this tied to nvmrc is that we lose the ability to make the node version a matrix and test on LTS and current versions of node. This would be useful to catch features dependent on specific package versions, but I don't think we're at that point yet. In practice, it's more important to us to test on different versions of terraform than node, so this is probably bike shedding.

Reply:
Yeah - I'm open to introducing a matrix of Node.js versions here, but I reckon we'd still want to be testing against the version used by nvm.

Reply:
Either way I'd prefer to leave that for another day/PR.

Reply:
Agreed